.github/workflows/core-vpc-development-deployment.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           terraform -chdir="terraform/environments/core-vpc" workspace select "core-vpc-${TF_ENV}"
 
           #RUN TERRAFORM PLAN
-          PLAN=`bash scripts/terraform-plan.sh terraform/environments/core-vpc -target=module.vpc | tee /dev/stderr | grep '^Plan: \|^No changes.'`
+          PLAN=`bash scripts/terraform-plan.sh terraform/environments/core-vpc -target=module.vpc -target=module.vpc_tgw_routing -target=module.core-vpc-tgw-routes -target=module.dns-zone -target=module.dns_zone_extend -target=module.vpc_nacls | tee /dev/stderr | grep '^Plan: \|^No changes.'`
```

Comment:
I think the script will treat each target block as a separate parameter unless we quote them all together